lib/source-map/mapping-list.js

```diff
@@ -0,0 +1,88 @@
+/* -*- Mode: js; js-indent-level: 2; -*- */
+/*
+ * Copyright 2014 Mozilla Foundation and contributors
+ * Licensed under the New BSD license. See LICENSE or:
+ * http://opensource.org/licenses/BSD-3-Clause
+ */
+if (typeof define !== 'function') {
+    var define = require('amdefine')(module, require);
+}
+define(function (require, exports, module) {
+
+  var util = require('./util');
+
+  /**
+   * Determine whether mappingB is after mappingA with respect to generated position.
+   */
+  function generatedPositionAfter(mappingA, mappingB) {
+    // Optimized for most common case
```

Comment:
To justify this unrolling, it provided a slightly over 2x speedup in my testing. Roughly corresponds to the `onlyCompareGenerated` flag in the util function.